sites/diverseeducation.com/server/templates/website-section/awards-honors/arthur-ashe/index.marko

```diff
@@ -56,7 +56,13 @@ $ const rootAlias = 'awards-honors/arthur-ashe';
         </a>
       </div>
     </marko-web-block>
-
+  </@section>
+  <@section|{ aliases }| modifiers=["first"]>
```

Comment:
Is this modifiers list necessary? Any other implementation of the rotation ad I'm seeing don't include it and this is likely included only for the `first` modifier to be added to the first ad on the page to prevent CLS issues and for targeting for thing like reskins or something to that effect.

Reply:
Removed two "first" modifiers